potato/create_task_cli.py

```diff
@@ -56,78 +56,112 @@ def get_initial_config():
     #
     config["base_html_template"] = "templates/base_template.html"
     config["header_file"] = "templates/header.html"
-        
+
     # This is where the actual HTML files will be generated
     config["site_dir"] = "potato/templates/"
 
     return config
 
 
-        
 def create_task_cli():
 
-    print('Welcome to the Potato annotation task creation process on the command line!')
-    print('This process will walk you through a series of questions about your task')
-    print('and generate a .yaml file that has the initial configuration for your')
-    print('task. You can edit this .yaml file at any time after the process to tweak')
-    print('the intructions, add fancier features, or even change the nature of the')
-    print('annotation. This process will only take a few minutes.')
-    print('')
-    print('If you don\'t yet know the answer to any of these questions you can leave')
-    print('the answer blank and edit your .yaml file later to fill in these details.')
-
-
-    print('')
-    print('First, some preliminary questions about how you want the server setup:')
-
```

Comment:
Any chance to revert this part? Some of the Black changes are kind of ... visually annoying. I'm on board with most of PEP-8 but not everything :)

Reply:
Done -- simply ran `black --line-length 100 ./`!